examples/user_guide/02_Data_Ingestion_with_Intake.ipynb

```diff
@@ -4,10 +4,389 @@
+   "source": [
+    "To get better insight into the data without loading it all in just yet, we can inspect the data using ``.to_dask()``"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training_dd = training.to_dask()\n",
+    "training_dd.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training_dd.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To get a full pandas.DataFrame object, use ``.read()`` to load in all the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training_df = training.read()\n",
+    "training_df.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**NOTE:** There are different items in these two info views which reflect what is knowable before and after we read all the data. For instance, it is not possible to know the ``shape`` of the whole dataset before it is loaded."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading multiple files\n",
+    "\n",
+    "In addition to allowing partitioned reading of files, intake lets the user load and concatenate data across multiple files in one command"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training = intake.open_csv(['../landsat5_training.csv', '../landsat8_training.csv'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training_df = training.read()\n",
+    "training_df.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**NOTE:** The length of the dataframe has increased now that we are loading multiple sets of training data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This can be more simply expressed as:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training = intake.open_csv('../landsat*_training.csv')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Sometimes, there is data encoded in a file name or path, that causes concatenated data to lose some important context. In this example, we lose the information about which version of landsat the training was done on. To maintain that information you can pass arguments into the csv reader directly like this:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training = intake.open_csv('../landsat*_training.csv', csv_kwargs=dict(include_path_column=True))\n",
```

Comment:
Mentioning `include_path_column` is good for completeness but I'm not sure it is strictly necessary: I suspect the format strings are always always more useful than the raw filenames. And won't a simple format string let you capture the whole filename if you want anyways?